common/src/main/java/com/gregtechceu/gtceu/data/recipe/misc/MetaTileEntityLoader.java

```diff
@@ -34,6 +34,7 @@ public class MetaTileEntityLoader {
     public static void init(Consumer<FinishedRecipe> provider) {
         CraftingComponent.initializeComponents();
 
+        VanillaRecipeHelper.addShapedRecipe(provider, true, "casing_ulv", GTBlocks.MACHINE_CASING_ULV.asStack(), "PPP", "PwP", "PPP", 'P', new UnificationEntry(TagPrefix.plate, GTMaterials.WroughtIron));
```

Comment:
why is there a duplicate ulv casing recipe?

Reply:
I probably accidentally copied that one it wasn't meant to be there